### ISSUE TITLE
Create unit tests to validate output code

### ISSUE THREAD

**Issue by wcmjunior** [ParallelME/compiler#20]
Unit tests must be created in order to evaluate compiler output code. Due to the complexity of the compiler and the creation of new features, it is getting tough to maintain code integrity without unit tests.
It is highly recommended that at least the following classes have all public methods tested:
- PMArrayTranslator
- PMBitmapImageTranslator
- PMHDRImageTranslator
- RSArrayTranslator
- RSBitmapImageTranslator
- RSHDRImageTranslator

In short, these classes represent the underlying translation system that is used to turn each user library class' intermediate representation into a valid code in Java and C.
